pkg/gatewayserver/observability.go

```diff
@@ -209,26 +218,44 @@ var gsMetrics = &messageMetrics{
 		},
 		[]string{protocol},
 	),
+	downlinkTxAckForwarded: metrics.NewContextualCounterVec(
+		prometheus.CounterOpts{
+			Subsystem: subsystem,
+			Name:      "downlink_tx_ack_forwarded_total",
+			Help:      "Total number of forwarded tx acknowledgments",
+		},
+		[]string{host, protocol},
+	),
+	downlinkTxAckDropped: metrics.NewContextualCounterVec(
```

Comment:
This is what `downlinkTxFailed` is for

Reply:
Same as `downlinkTxSucceeded` vs `downlinkTxAckForwarded`